custom/icds_reports/sqldata/exports/children.py

```diff
@@ -53,95 +52,98 @@ def columns(self):
                 'Weighing efficiency (in month)',
                 percent,
                 [
-                    SumColumn('nutrition_status_weighed', filters=self.filters + [
-                        NOT(EQ('age_tranche', 'age_72'))
-                    ]),
-                    SumColumn('wer_eligible', filters=self.filters + [
-                        NOT(EQ('age_tranche', 'age_72'))
-                    ])
+                    SumWhen(
+                        whens={"age_tranche != :age_72": 'nutrition_status_weighed'}, else_=0,
+                        alias='nutrition_status_weighed'
+                    ),
+                    SumWhen(
+                        whens={"age_tranche != :age_72": 'wer_eligible'}, else_=0,
+                        alias='wer_eligible'
+                    )
                 ],
                 slug='percent_weight_efficiency'
             ),
             AggregateColumn(
                 'Height measurement efficiency (in month)',
                 percent,
                 [
-                    SumColumn(
-                        'height_measured_in_month',
-                        filters=self.filters + [NOT(EQ('age_tranche', 'age_72'))],
-                        alias='height_measured_in_month_all'
+                    SumWhen(
+                        whens={"age_tranche != :age_72": 'height_measured_in_month'}, else_=0,
```

Comment:
should this be using `hfa_recorded_in_month_column(self.beta)` like the others in this file?